test/resources/must-equal-strict-doc-definitions.js

```diff
@@ -0,0 +1,133 @@
+function() {
```

Comment:
As noted in the pull request description, this file borrows from `must-equal-doc-definitions.js`.